arcee/api.py

```diff
@@ -33,17 +37,55 @@ def upload_qa_pairs(qa_set: str, qa_pairs: List[Dict[str, str]]) -> Dict[str, st
         Dict[str, str]: The response from the make_request call.
     """
     if len(qa_pairs) > 2000:
-        raise Exception("You can only upload 2000 QA pairs at a time")  
+        raise Exception("You can only upload 2000 QA pairs at a time")
 
     qa_list = []
     for qa in qa_pairs:
-        if "question" not in qa or "answer" not in qa:
+        if prompt_column not in qa.keys() or completion_column not in qa.keys():
             raise Exception("Each QA pair must have a 'question' and an 'answer' key")
 
-        qa_list.append({"question": qa["question"], "answer": qa["answer"]})
+        qa_list.append({"question": qa[prompt_column], "answer": qa[completion_column]})
 
     data = {"qa_set_name": qa_set, "qa_pairs": qa_list}
-    return make_request("post", Route.alignment+"/qaUpload", data)
+    return make_request("post", Route.alignment + "/qaUpload", data)
+
+def chunk_list(lst, chunk_size):
+    """Yield successive n-sized chunks from lst."""
+    for i in range(0, len(lst), chunk_size):
+        yield lst[i:i + chunk_size]
+
+def upload_instructions_from_csv(qa_set: str, csv_path: str, prompt_column: str = "prompt", completion_column: str = "completion", batch_size: int = 200) -> None:
+    """
+    Upload QA pairs from a CSV file to a specific QA set.
+
+    Args:
```

Comment:
Can you add docs for the `prompt_column` and `completion_column`?

Reply:
It would be good to tell the user that these column names will be propagated to the system.